src/main/java/com/starkbank/CorporateHolder.java

```diff
@@ -0,0 +1,860 @@
+         * after [string, default null]: date filter for objects created only after specified date. ex: "2020-03-10"
+         * before [string, default null]: date filter for objects created only before specified date. ex: "2020-03-10"
+         * types [list of strings, default null]: filter retrieved objects by types. ex: "created" or "blocked"
+         * holderIds [list of strings, default null]: list of CorporateHolder ids to filter retrieved objects. ex: ["5656565656565656", "4545454545454545"]
+         * ids [list of strings, default null]: list of ids to filter retrieved objects. ex: ["5656565656565656", "4545454545454545"]
+         * @param user [Organization/Project object, default null]: Organization or Project object. Not necessary if starkbank.Settings.user was set before function call
+         * <p>
+         * Return:
+         * @return CorporateHolder.Log.Page object:
+         * CorporateHolder.Log.Page.logs: list of CorporateHolder.Log objects with updated attributes
+         * CorporateHolder.Log.Page.cursor: cursor to retrieve the next page of CorporateHolder.Log objects
+         * @throws Exception error in the holder
+         */
+        public static CorporateHolder.Log.Page page(Map<String, Object> params, User user) throws Exception {
+            com.starkbank.utils.Page page = Rest.getPage(data, params, user);
+            List<CorporateHolder.Log> logs = new ArrayList<>();
+            for (SubResource log: page.entities) {
+                logs.add((CorporateHolder.Log) log);
+            }
+            return new Page(logs, page.cursor);
+        }
+    }
+
+    private List<CorporateHolder.Permission> parsePermissions(List<Object> permissions) throws Exception {
+        if (permissions == null)
+            return null;
+
+        List<CorporateHolder.Permission> parsed = new ArrayList<>();
+        if (permissions.size() == 0 || permissions.get(0) instanceof CorporateHolder.Permission) {
+            for (Object permission : permissions) {
+                parsed.add((CorporateHolder.Permission) permission);
+            }
+            return parsed;
+        }
+
+        for (Object permission : permissions) {
+            CorporateHolder.Permission permissionObject = new CorporateHolder.Permission((Map<String, Object>) permission);
+            parsed.add(permissionObject);
+        }
+        return parsed;
+    }
+
+    public final static class Permission extends SubResource{
+        public String ownerId;
+        public String ownerType;
+        public String ownerEmail;
+        public String ownerName;
+        public String ownerPictureUrl;
+        public String ownerStatus;
+        public String created;
+
+
+        /**
+         * CorporateHolder.Permission object
+         * <p>
+         * The CorporateHolder.Permission object modifies the behavior of CorporateHolder objects when passed as an argument upon their creation.
```

Comment:
Acho que a descrição correta é essa: 

`Permission object represents access granted to an user for a particular cardholder`